test/spawn.jl

```diff
@@ -430,7 +430,7 @@ end
 if is_unix()
     let ps = Pipe[]
         try
-            for i = 1:100_000
+            for i = 1 : 100 * parse(Int, readchomp(`sh -c 'ulimit -n'`))
```

Comment:
any idea how to get that to happen? even as root I'm having a hard time setting it to anything above 1048576